src/dbnode/client/config.go

```diff
@@ -237,6 +237,13 @@ type ConfigurationParameters struct {
 	// EncodingOptions is an optional argument when
 	// constructing a client from configuration.
 	EncodingOptions encoding.Options
+
+	// AllowEmptyInitialNamespaceRegistry is an optional argument when
+	// constructing a client from configuration. If enabled,
+	// namespaceInitializer from client.Options will not
+	// wait indefinitely until non-empty namespace registry
+	// is received.
+	AllowEmptyInitialNamespaceRegistry bool
```

Comment:
Do we want to add another config field? When would we still want the old blocking behavior if we've moved to this new model?

Reply:
Synced offline. Going to remove this from the client.